libraries/core/lib/search/metaheuristics/evolutionary/MOSAFamily.ts

```diff
@@ -72,10 +72,10 @@ export class MOSAFamily<T extends Encoding> extends EvolutionaryAlgorithm<T> {
       return; // the search should end
 
     // non-dominated sorting
-    getUserInterface().debug(
```

Comment:
These should be put in the logger and not in the UI

Reply:
Fixed in #238.